src/Components/MovieList/index.tsx

```diff
@@ -15,8 +14,10 @@ import {
     fetchMoviesSuccess,
 } from './actions'
 import * as Selectors from './selectors'
+import { APP_INIT_STORE_STATE } from 'Utils/constants'
 
 const MovieList: React.FC = () => {
+    const [isFirstRendering, setIsFirstRendering] = useState(true)
```

Comment:
not an error, but it will be better, to use `useRef` for this logic to reduce some unnecessary rerenders

Reply:
Thank you.
Using ref now